lib/bowling_score.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+# BowlingScore used to take a single string showing the score of each throw of
+# the game as input and calculate the total score
+class BowlingScore
+  attr_accessor :all_throws
+
+  def initialize(all_throws, full_game = false)
+    @all_throws = all_throws.split('')
```

Comment:
Fair play mate I can understand why you've gone on this approach. It would be good to potentially try out on a separate branch as an extension if you get some time. Don't worry too much about it for now though!